### ISSUE TITLE
adding learning rate to optimizers

### ISSUE THREAD

**Issue by mikkokotila** [autonomio/core-module#118]
We should agree on a best way to add optimizer learning rates as a parameter to train().

`https://keras.io/optimizers/`

**Comment by konchokdolma** [comment]
I have added changes with the learning rate parameters to a daily-dev. As a default it is 'auto', which leaves optimizer as a string. If it is not 'auto' and model is mlp, we are calling it using get_method in utils.
For regression and lstm models it is easier, as optimizer for now is fixed.
